### PR TITLE
Feature: jenkins approve

### PR DESCRIPTION
**Pull request by mateuszszulc**

### CODE REVIEW

**Review by mszulc-csod — APPROVED** [review]
Beautiful PR.

**Comment by mszulc-csod** [comment]
Open conversation

File: package.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "name": "playwright-travis-jenkins",
+  "version": "1.0.0",
```
Comment (mszulc-csod) [inline package.json:3]: Start conversation

Comment (mateuszszulc) [reply package.json:3]: OK

**Pull request merged**